oxide-auth/src/code_grant/accesstoken.rs

```diff
@@ -563,13 +563,13 @@ pub struct PrimitiveError {
 /// addition this enforces backend specific behaviour for obtaining or handling the access error.
 #[derive(Clone)]
 pub struct ErrorDescription {
-    pub(crate) error: AccessTokenError,
+    pub error: AccessTokenError,
 }
```

Comment:
This will disallow adding further fields to the description in compatible releases. Intentional?

Reply:
Huh, unintentional since reverting the change seems to have no impact on the tests.

Reply:
Intentional, needed for `client_credentials`. I just added a `ErrorDescription::new` function instead.